src/commands/improveFileNamesInContentDirectory.ts

```diff
@@ -0,0 +1,79 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { renameFile } from '../utils/fileOperations';
+import { supportedFileTypes } from './supportedFileTypes';
+import * as fs from 'fs';
+import { renameFileViaApi } from '../api/renamerApi';
+import { RenameDirPayload } from '../types';
+
+export const improveFileNamesInContentDirectory = async () => {
+    const folderUri = await vscode.window.showOpenDialog({
```

Comment:
can we improve this to donot open the dialog box to select the folder structure instead it should select the currently open folder . Probably we can solve this ( if we are planning to) in a separate PR .